docs/REFERENCE.md

```diff
@@ -1853,6 +1853,7 @@ chezmoi provides the following automatically-populated variables:
 | -------------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------- |
 | `.chezmoi.arch`            | `string`   | Architecture, e.g. `amd64`, `arm`, etc. as returned by [runtime.GOARCH](https://pkg.go.dev/runtime?tab=doc#pkg-constants).      |
 | `.chezmoi.args`            | `[]string` | The arguments passed to the `chezmoi` command, starting with the program command.                                               |
+| `.chezmoi.executable` | `string` | The path to the `chezmoi` executable, if available. |
```

Comment:
Please align the `|`s here.

Reply:
I just had to disable the auto generation of toc on save:

https://github.com/twpayne/chezmoi/pull/1618/files#diff-24ad71c8613ddcf6fd23818cb3bb477a1fb6d83af4550b0bad43099813088686R21-R22

And I also removed a deprecated configuration (the terminal).

Reply:
Ops, I initially pushed with `"markdown.extension.toc.updateOnSave": true` but in fact it was meant to be `false`, as they TOCs seems to be generated by the website itself.